23/README.md

```diff
@@ -0,0 +1,283 @@
+We also run other tests using a customized version of [`tm-bench`][tm:tm-bench].
+
+Another thing to consider is that BigchainDB is a multi-asset blockchain. With BigchainDB, users can create their own custom assets. Each new asset is identified by a unique value, that is the `id` of the `CREATE` transaction that minted it. Assets have their own unique history or—in more precise terms—directed acyclic graph. In order to check if a transaction is a double spend, we can safely ignore all transactions related to other assets. This means that when BigchainDB has to validate a block, it can parallelize validation, exploiting all cores in the current machine (more details to come). To understand if we could get a performance boost out of this, we integrated a new experimental feature that enables parallel validation of transactions. This feature is a proof of concept to show that higher throughput is realizable. Note: this feature skips also *Check Tx*, used by Tendermint to check a transaction before storing it in the mempool. BigchainDB validates transactions as soon as they are posted to the HTTP API, so *Check Tx* in this case is redundant. We must mention that *Check Tx* is used also to validate transactions coming from the other nodes of the network, so disabling it might allow byzantine actors to push invalid transactions to the mempool (handling invalid transactions is [actively discussed in the Tendermint issue tracker][tm:invalid-txs]). Even if an invalid transaction makes it to the mempool, it will be always discarded during the *Deliver Tx* phase, so this experimental feature will still maintain a correct state of the application. Parallel validation is an optimistic approach that will be eventually developed for specific production use cases.
+
+## Methods
+<!--
+METHODS. The methods section will help you determine exactly how the authors performed the experiment.
+
+The methods describes both specific techniques and the overall experimental strategy used by the scientists. Generally, the methods section does not need to be read in detail. Refer to this section if you have a specific question about the experimental design.
+-->
+
+The performance tests were run on the Microsoft Azure cloud infrastructure. For the BigchainDB nodes, we choose compute optimized virtual machines from the [Fsv2 series][azure:fsv2], specifically *Standard F32s v2* (32 vCPUs, 64GiB memory, SSD drives with estimated IOPS limit of 5000). For the machine generating the workload, we used a *Standard F8s v2* (8 vCPUs, 16GiB memory). The virtual machines were all in the same datacenter.
```

Comment:
I checked the linked chart about the Fsv2 series VMs and it says that for Standard_F32s_v2 VMs:

Max cached and temp storage throughput: IOPS / MBps (cache size in GiB) is 64000 (512)

and

Max NICs / Expected network bandwidth (Mbps) is 8 / 14,000

Reply:
About the IOPS of the disk, I used the data coming from portal.azure.com... Not sure why they have two different numbers, check this screenshot:
![image](https://user-images.githubusercontent.com/134680/45546496-c72d5780-b81d-11e8-85e1-93669e74c8cb.png)


I'll make the suggested changes.